Use real data when rendering service manual frontend guides

tadast opened the pull request:
We had a couple of placeholders on the guides page. Real data is being published now, so we can display it on the page.

KushalP commented:
There was one small thing I picked up about how general `Publisher` is, but the rest of this looks good.

Once you've changed that, I'm happy with this PR.

tadast commented:
@KushalP I have renamed the field and added another commit to show `<title>`. The build should fix itself once we merge https://github.com/alphagov/govuk-content-schemas/pull/150
